lib/server/server-google-pubsub.ts

```diff
@@ -134,9 +134,15 @@ export class GooglePubSubTransport extends Server implements CustomTransportStra
 
     /**
      * Resolve subscriptions and create them if `createSubscription` is true
+     * Basic logic is as follows:
+     *   If a subscription name is provided and that subscription exists, use it
+     *   If a subscription name is provided and it does not exist, attempt to create it and use it
+     *       If a topic name is provided and that topic exists, create our subscription
+     *       If a topic name was not provided OR the topic does not exist, throw an error
```

Comment:
```suggestion
     * @remarks
     * Basic logic is as follows:
     *   If a subscription name is provided and that subscription exists, use it
     *   If a subscription name is provided and it does not exist, attempt to create it and use it
     *       If a topic name is provided and that topic exists, create our subscription
     *       If a topic name was not provided OR the topic does not exist, throw an error
```

TSDoc standard specifies that the more detailed part of the description falls under [remarks](https://tsdoc.org/pages/tags/remarks/)

Reply:
Ah, I'll update